x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_creation/threshold_rule.cy.ts

```diff
@@ -130,4 +140,21 @@ describe('Threshold rules', { tags: ['@ess', '@serverless'] }, () => {
     cy.get(ALERTS_COUNT).should(($count) => expect(+$count.text().split(' ')[0]).to.be.lt(100));
     cy.get(ALERT_GRID_CELL).contains(rule.name);
   });
+
+  it('Creates a new threshold rule with suppression enabled', () => {
+    selectThresholdRuleType();
+
+    enablesAndPopulatesThresholdSuppression(5, 'h');
+    fillDefineThresholdRuleAndContinue(rule);
+
+    fillAboutRuleMinimumAndContinue(rule);
+    skipScheduleRuleAction();
+    createRuleWithoutEnabling();
+    openRuleManagementPageViaBreadcrumbs();
+    goToRuleDetailsOf(rule.name);
+
+    cy.get(DEFINITION_DETAILS).within(() => {
+      getDetails(SUPPRESS_FOR_DETAILS).should('have.text', '5h');
```

Comment:
I see that in other rule creation tests we assert more details as well as check that the number of alerts generated are the correct ones. Is there any reason why we are checking just this?